session/container.go

```diff
@@ -147,6 +147,14 @@ func (x *Container) UnmarshalJSON(data []byte) error {
 //
 // See also [Container.VerifySignature], [Container.SignedData].
 func (x *Container) Sign(signer user.Signer) error {
+	x.issuer = signer.UserID()
+	x.issuerSet = true
+	return x.SetSignature(signer)
+}
+
+// SetSignature allows to sign Container like [Container.Sign] but without
+// issuer setting.
+func (x *Container) SetSignature(signer neofscrypto.Signer) error {
```

Comment:
Isn't "static" signer doing the same? It can have any `UserID()`.

Reply:
no: `user.Signer` will set the user as an issuer (could be any but still fixed), while this method doesnt touch issuer at all

Reply:
The end result is the same to me. We tell people to use static signer when they need to, which is equal to `SetIssuer/SetSignature`. Can we explain the difference and use cases to people? I don't see any other use but tests for it (where you may need an old token without issuer at all), but likely tests can handle it internally.

Reply:
OK, we can keep it. My primary concern here probably is consistency with other types. We currently provide `SetSignature` for objects in SDK, but otherwise it's an API thing. At the same time it probably doesn't hurt either.